### PR TITLE
Chat tests

### PR DESCRIPTION
**Pull request by BartDevTWG**
## Description

**Does this PR introduces a breaking change?**

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate).

### CODE REVIEW

**Review by pavelsof — APPROVED** [review]
Tests! :raised_hands:

File: src/chat/components/InputToolbar.tsx [hunk 1]
```diff
@@ -86,7 +90,10 @@ export default function InputToolbar({ onSubmit }: InputToolbarProps) {
             disabled={!message}
             onClick={handleSubmit}
           >
-            Send
+            {intl.formatMessage({
```
Comment (alex-blair) [inline src/chat/components/InputToolbar.tsx:93]: Nice replacing the hard-coded string!

File: src/chat/__tests__/Inbox.test.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      },
+    });
+    expect(queryByTestId(CHAT_WRAPPER)).toBeInTheDocument();
+  });
+  it('fetchs chat auth token, then connects to socket', async () => {
+    renderWithRedux(<InboxWithProviders />);
+    await waitFor(() => {
+      expect(mockGetChatAuthorizationToken).toBeCalledTimes(1);
+    });
+    // TODO: test socket
```
Comment (alex-blair) [inline src/chat/__tests__/Inbox.test.tsx:39]: Is this TODO still needed or can it be removed?

**Review by alex-blair — APPROVED** [review]
A whole PR full of tests! Amazing! 🙌

File: src/chat/__tests__/Inbox.test.tsx (hunk 2 above)
Comment (BartDevTWG) [reply src/chat/__tests__/Inbox.test.tsx:39]: I'm still looking for a way to mock socket and channels connection

**Pull request merged**